src/api/flaskr/service/profile/funcs.py

```diff
@@ -137,13 +137,13 @@ def save_user_profile(
         )
         db.session.add(user_profile)
     if profile_key in PROFILES_LABLES:
```

Comment:
```suggestion
    if profile_key in PROFILES_LABELS:
```
Should change all the `PROFILES_LABLES` to `PROFILES_LABELS`?